### PR TITLE
Update release notes draft to version v1.32.0-alpha.2

### PR DESCRIPTION
**Pull request by rayandas**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the Release Notes Draft to k/k v1.32.0-alpha.2

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

File: releases/release-1.32/release-notes/maps/pr-125923-map.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+pr: 125923
+releasenote:
+  text: 'Fixed a bug where the kubelet ephemerally failed with `failed to initialize
+    top level QOS containers: root container [kubepods] doesn''t exist`, due to the
+    cpuset cgroup being deleted on v2 with systemd cgroup manager.'
```
Comment (npolshakova) [inline releases/release-1.32/release-notes/maps/pr-125923-map.yaml:5]: Does the extra `''` break the yaml?

```suggestion
  text: 'Fixed a bug where the kubelet ephemerally failed with `failed to initialize
    top level QOS containers: root container [kubepods] doesn't exist`, due to the
    cpuset cgroup being deleted on cgroup v2 with systemd cgroup manager.'
```

File: releases/release-1.32/release-notes/maps/pr-127611-map.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+pr: 127611
+releasenote:
+  text: Kubernetes was built with go 1.23.1.
```
Comment (karenhchu) [inline releases/release-1.32/release-notes/maps/pr-127611-map.yaml:3]:
```suggestion
  text: Kubernetes was built with Go 1.23.1.
```

File: releases/release-1.32/release-notes/maps/pr-128110-map.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+pr: 128110
+releasenote:
+  text: Kubernetes was built with go 1.23.2.
```
Comment (karenhchu) [inline releases/release-1.32/release-notes/maps/pr-128110-map.yaml:3]:
```suggestion
  text: Kubernetes was built with Go 1.23.2.
```

**Review by karenhchu — COMMENTED** [review]
few nit picks, otherwise LGTM

**Comment by satyampsoni** [comment]
/lgtm
Thanks @rayandas.

**Comment by fsmunoz** [comment]
Thanks !

/lgtm
/approve

**Pull request merged**